src/auto-evo/selection_pressure/RootPressure.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace AutoEvo;
+
+using System;
+
+public class RootPressure : SelectionPressure
+{
+    // Needed for translation extraction
+    // ReSharper disable ArrangeObjectCreationWhenTypeEvident
+    public static readonly LocalizedString Name = new LocalizedString("ROOT_PRESSURE");
+
+    // ReSharper restore ArrangeObjectCreationWhenTypeEvident
+
+    public RootPressure() : base(1, [
+        new RemoveOrganelle(_ => true),
+        new AddOrganelleAnywhere(_ => true),
+        new AddOrganelleAnywhere(organelle => organelle.InternalName == "nucleus"),
+    ])
+    {
+    }
+
+    public override float Score(Species species, SimulationCache cache)
+    {
+        return 1;
+    }
+
+    public override float GetEnergy()
+    {
+        return 0;
+    }
+
+    public override IFormattable GetDescription()
+    {
+        // This shouldn't be called on 0 energy pressures
```

Comment:
Why is this comment here? This sounds more like what the base class should have.